jit/jit-apply.c

```diff
@@ -239,8 +239,6 @@ static void
 jit_apply_builder_get_return(jit_apply_builder *builder, void *rv,
 			     jit_type_t type, jit_apply_return *result)
 {
-	unsigned int size;
```

Comment:
This variable is used when HAVE_JIT_BUILTIN_APPLY_STRUCT_RETURN is not defined (take a look at the JIT_TYPE_STRUCT case in this function).